content/5_Plat/Geo_Pri.mdx

```diff
@@ -379,7 +374,7 @@ int main() {
 	long long boundary_points = 0;
 	for (int i = 0; i < n; i++) {
 		Point diff = points[i + 1] - points[i];
-		int g = gcd(abs(diff.x), abs(diff.y));
+		int g = __gcd(abs(diff.x), abs(diff.y));
```

Comment:
really sorry but could you use [this gcd](https://en.cppreference.com/w/cpp/numeric/gcd) instead?

Reply:
Done